packages/protocol-kit/tests/e2e/createSafeDeploymentTransaction.test.ts

```diff
@@ -1,5 +1,6 @@
 import chai from 'chai'
 import chaiAsPromised from 'chai-as-promised'
+import { keccak256 } from 'ethereumjs-util'
```

Comment:
@DaniSomoza this lib is deprecated at your PR, could you recommend an alternative here?

Reply:
https://github.com/paulmillr/noble-hashes

Reply:
Thx, I updated the PR to use `@noble/hashes` now.